tpm2/error.go

```diff
@@ -0,0 +1,325 @@
+	RcNVUnavailable         = 0x23
+)
+
+var warnMgs = map[RcWarn]string{
+	RcContextGap:     "gap for context ID is too large",
+	RcObjectMemory:   "out of memory for object contexts",
+	RcSessionMemory:  "out of memory for session contexts",
+	RcMemory:         "out of shared object/session memory or need space for internal operations",
+	RcSessionHandles: "out of session handles",
+	RcObjectHandles:  "out of object handles",
+	RcLocality:       "bad locality",
+	RcYielded:        "the TPM has suspended operation on the command; forward progress was made and the command may be retried",
+	RcCanceled:       "the command was canceled",
+	RcTesting:        "TPM is performing self-tests",
+	RcReferenceH0:    "the 1st handle in the handle area references a transient object or session that is not loaded",
+	RcReferenceH1:    "the 2nd handle in the handle area references a transient object or session that is not loaded",
+	RcReferenceH2:    "the 3rd handle in the handle area references a transient object or session that is not loaded",
+	RcReferenceH3:    "the 4th handle in the handle area references a transient object or session that is not loaded",
+	RcReferenceH4:    "the 5th handle in the handle area references a transient object or session that is not loaded",
+	RcReferenceH5:    "the 6th handle in the handle area references a transient object or session that is not loaded",
+	RcReferenceH6:    "the 7th handle in the handle area references a transient object or session that is not loaded",
+	RcReferenceS0:    "the 1st authorization session handle references a session that is not loaded",
+	RcReferenceS1:    "the 2nd authorization session handle references a session that is not loaded",
+	RcReferenceS2:    "the 3rd authorization session handle references a session that is not loaded",
+	RcReferenceS3:    "the 4th authorization session handle references a session that is not loaded",
+	RcReferenceS4:    "the 5th authorization session handle references a session that is not loaded",
+	RcReferenceS5:    "the 6th authorization session handle references a session that is not loaded",
+	RcReferenceS6:    "the 7th authorization session handle references a session that is not loaded",
+	RcNVRate:         "the TPM is rate-limiting accesses to prevent wearout of NV",
+	RcLockout:        "authorizations for objects subject to DA protection are not allowed at this time because the TPM is in DA lockout mode",
+	RcRetry:          "the TPM was not able to start the command",
+	RcNVUnavailable:  "the command may require writing of NV and NV is not current accessible",
+}
+
+// Indexes for arguments, handles and sessions.
+const (
+	Rc1 RcIndex = 0x01
+	Rc2         = 0x02
+	Rc3         = 0x03
+	Rc4         = 0x04
+	Rc5         = 0x05
+	Rc6         = 0x06
+	Rc7         = 0x07
+	Rc8         = 0x08
+	Rc9         = 0x09
+	RcA         = 0x0A
+	RcB         = 0x0B
+	RcC         = 0x0C
+	RcD         = 0x0D
+	RcE         = 0x0E
+	RcF         = 0x0F
+)
+
+type Error struct {
```

Comment:
Add doc comment https://github.com/golang/go/wiki/CodeReviewComments#initialisms
Here and for all other exported types.

You can leave `Error` methods without a comment